packages/docs/src/examples/selection-controls/labelSlot.vue

```diff
@@ -1,28 +1,44 @@
 <template>
   <v-container fluid>
     <v-checkbox v-model="checkbox">
-      <div slot="label">
-        I agree that
-        <v-tooltip bottom>
-          <a slot="activator" target="_blank" href="http://vuetifyjs.com" @click.stop>Vuetify</a>
-          Opens in new window
-        </v-tooltip>
-        is awesome
-      </div>
+      <template v-slot:label>
+        <div>
+          I agree that
+          <v-tooltip bottom>
+            <template v-slot:activator="{ on: tooltip }"></template>
+            <a
+              target="_blank"
+              href="http://vuetifyjs.com"
+              @click.stop
+              v-on="tooltip"
```

Comment:
this is throwing errors 
![image](https://user-images.githubusercontent.com/9064066/52866097-966e8880-3103-11e9-839c-564e1325383c.png)